### ISSUE TITLE
Unhandled rejection TypeError: _behavior is undefined

### ISSUE THREAD

**Issue by Tim-Erwin** [aurelia/web-components#4]
**I'm submitting a bug report**

* **Library Version:**
0.1.0

**Please tell us about your environment:**
* **Operating System:**
OSX 10.13.4

* **Node Version:**
8.9.4

* **NPM Version:**
5.8.0

* **Browser:**
Firefox 59

* **Language:**
TypeScript

**Current behavior:**
I get a `Unhandled rejection TypeError: _behavior is undefined` when starting the app.

**Steps to Reproduce**

    au new comptest
    2
    <ENTER>
    <ENTER>
    cd comptest/
    npm install aurelia-web-components
    au generate element my-component

    ... copy & paste the contents from the *Usage* part of the docs to main.ts

    au run --watch

**Complete Stacktrace**

    Unhandled rejection TypeError: _behavior is undefined
        getObserver@webpack-internal:///./node_modules/aurelia-templating/dist/native-modules/aurelia-templating.js:3635:9
        _configureDescriptor/descriptor.get@webpack-internal:///./node_modules/aurelia-templating/dist/native-modules/aurelia-templating.js:3697:14
        _createWebComponentClassFromBehavior/<@webpack-internal:///./node_modules/aurelia-web-components/dist/commonjs/aurelia-web-components.js:169:11
        _createWebComponentClassFromBehavior@webpack-internal:///./node_modules/aurelia-web-components/dist/commonjs/aurelia-web-components.js:168:5
        registerBehavior@webpack-internal:///./node_modules/aurelia-web-components/dist/commonjs/aurelia-web-components.js:47:27
        registerAllGlobalElements/<@webpack-internal:///./node_modules/aurelia-web-components/dist/commonjs/aurelia-web-components.js:42:14
        registerAllGlobalElements@webpack-internal:///./node_modules/aurelia-web-components/dist/commonjs/aurelia-web-components.js:40:12
        configure/<@webpack-internal:///main:34:9
    From previous event:
        configure@webpack-internal:///main:30:5
        config/<@webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:101:14
    From previous event:
        config@webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:96:12
        bootstrap/p<@webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:132:12
    From previous event:
        bootstrap@webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:131:11
        run/<@webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:118:7
    From previous event:
        run@webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:113:10
        @webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:138:16
        @webpack-internal:///./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js:2:29
        ./node_modules/aurelia-bootstrapper/dist/native-modules/aurelia-bootstrapper.js@http://localhost:8080/app.1e876d838a5e87e3a79e.bundle.js:94:1
        __webpack_require__@http://localhost:8080/app.1e876d838a5e87e3a79e.bundle.js:20:12
        [0]@http://localhost:8080/app.1e876d838a5e87e3a79e.bundle.js:617:18
        __webpack_require__@http://localhost:8080/app.1e876d838a5e87e3a79e.bundle.js:20:12
        @http://localhost:8080/app.1e876d838a5e87e3a79e.bundle.js:69:18
        @http://localhost:8080/app.1e876d838a5e87e3a79e.bundle.js:1:11

I tried debugging this but somehow I cannot get a good starting point. I'd be glad to help out but I'd need a hint or two :)

**Comment by EisenbergEffect** [comment]
@Tim-Erwin Thanks for trying this out! Can you zip up your project and attach it here? That would help me in debugging and getting a fix out.

**Comment by Tim-Erwin** [comment]
Sure, I removed `node_modules`. [comptest.zip](https://github.com/aurelia/web-components/files/1967859/comptest.zip) Thanks for looking into this so quickly.

**Comment by EisenbergEffect** [comment]
Ok, I've found two bugs. I'll try to get those fixed shortly and update the release. You are literally the first person to try this library out :) Thanks for your patience. There are likely more issues but I'm happy to work with you to resolve them as you find them.

**Comment by EisenbergEffect** [comment]
Ok, I've just released 0.2.0 which I tested against your project in Chrome. I've made an adjustment to your code to place an `au-app` element in your html file directly and ensure that it's globally registered with Aurelia so that the web components plugin picks it up. Your updated app (minus node_modules) is attached here. Thanks for helping out!

[comptest.zip](https://github.com/aurelia/web-components/files/1969330/comptest.zip)
